pkg/packagekit/updates.jsx

```diff
@@ -245,6 +266,39 @@ function UpdatesList(props) {
     );
 }
 
+function UpdateHistory(props) {
+    if (!props.history)
+        return null;
+
+    var now = Date.now();
+
+    function formatHeading(time) {
+        if (time)
+            return cockpit.format(_("The following packages were updated $0 ago:"), moment.duration(now - time).humanize());
+        return _("The following packages were recently updated:");
+    }
+
+    function formatPkgs(pkgs) {
+        var names = Object.keys(pkgs);
+        names.sort();
+        names.shift(); // remove _time
+        return names.map(n => <Tooltip tip={n + " " + pkgs[n]}><li>{n}</li></Tooltip>);
```

Comment:
please add spaces in complex `{ }`:
`return names.map(n => <Tooltip tip={ n + " " + pkgs[n] }><li>{n}</li></Tooltip>);`